packages/blocks/cloudzero/src/lib/auth.ts

```diff
@@ -0,0 +1,9 @@
+import { BlockAuth } from '@openops/blocks-framework';
+
+export const cloudzeroAuth = BlockAuth.SecretAuth({
```

Comment:
Add a markdown please how to generate the auth.

Do you know also if the auth token has any expiry?

Reply:
from my understanding no https://docs.cloudzero.com/reference/authorization